__tests__/app.test.js

```diff
@@ -45,25 +37,143 @@ describe('/api', () => {
     });
 });
 
-describe('/api/articles/:article_id', () => {
+describe('GET: /api/articles/:article_id', () => {
     test('GET: 200 an article object with all associated properties', () => {
         return request(app)
         .get('/api/articles/1')
         .expect(200)
         .then((response) => {
-            const {rows} = response.body
-            rows.forEach(row => {
-                expect(row).toMatchObject({
+            const {article} = response.body
+            expect(article.article_id).toBe(1)
+            expect(article).toMatchObject({
+                created_at: expect.any(String)
+            })
+        })
+    })   
+});
+
+describe('GET: /api/articles', () => {
+    test('GET: 200 returns an array of all objects within articles without body property, sorted in descending order by creation date', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const {articles} = response.body
+            expect(articles.length).toBe(37)
+            articles.forEach(article => {
+                expect(article).toMatchObject({
                     title: expect.any(String),
                     topic: expect.any(String),
                     author: expect.any(String),
-                    body: expect.any(String),
                     created_at: expect.any(String),
                     votes: expect.any(Number),
-                    article_img_url:
-                    expect.any(String),
+                    article_img_url: expect.any(String),
+                    comments_count: expect.any(String)
                 })
+                const articleBody = {body: expect.any(String)}
+                expect(articles).toEqual(expect.not.objectContaining(articleBody))
+                expect(articles).toBeSorted({ descending: true });
             })
         })
+    });
+});
+
+describe('GET: /api/articles/:article_id/comments', () => {
+    test('GET: 200 returns all comments associated with a specific article_id in ascending order based on creation date', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            const {comments} = response.body
+            comments.forEach(comment => {
+                expect(comment).toMatchObject({
+                    comment_id : expect.any(Number),
+                    body: expect.any(String),
+                    article_id: 1,
+                    author: expect.any(String),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String)
+                    })
+                })
+                expect(comments).toBeSorted({ descending: false });
+            })
+    });
+    test('POST:201', () => {
```

Comment:
As there are more than one type of 201 possible, a more verbose description would be great here